airflow/providers/amazon/aws/operators/redshift_cluster.py

```diff
@@ -252,6 +257,16 @@ def execute(self, context: Context):
             self.master_user_password,
             params,
         )
+        if self.deferrable:
```

Comment:
I dont see this new branch being tested in unit test. Do you test it somewhere?

Reply:
The unit test for this branch is fairly trivial, but, you're correct, it should still be included. Adding it now. Thanks!